make.js

```diff
@@ -1021,8 +1031,22 @@ CLI.gentask = function() {
             console.log(`Running \"npm install\" command in ${taskPath}`);
             run(`npm install`);
             cd(__dirname);
+
+            // copy package.json, package-lock.json and npm-shrinkwrap.json files from generated tasks to the Tasks\taskname\_buildConfig\nodeversion folder.
+            const fileNames = ['package.json', 'package-lock.json', 'npm-shrinkwrap.json'];
```

Comment:
maybe better to reuse [this function ](https://github.com/microsoft/azure-pipelines-tasks/blob/master/make-util.js#L1902) it's doing the same thing but with the build command(probably we need to add npm-shrinkwrap.json file to allowedFilesToCopy)?

Reply:
Files are not getting copied after adding syncGeneratedFilesWrapper here, probably something different between logics here. I'll keep the logics here for consistency and get back to this.